Add gwasm workshop video (#33)
golemfactory/golem-wiki: About/Knowledge-base.md

### Demos

##### Building on top of Golem with gWASM

> Have you missed our workshop at Devcon5 in Osaka? Fear not! We're pleased to give you the very workshop re-recorded at our office at the heart of Poland in Warsaw. Learn about Golem, our meta-use case gWasm, and once you digested all the information... please remember to put what you learn into good use and start coding for gWasm!

Reviewer: ```suggestion
Did you miss our workshop at Devcon5 in Osaka? Fear not! We're pleased to give you the very workshop re-recorded at our office at the heart of Poland in Warsaw. Learn about Golem, our meta-use case gWasm, and once you have digested all the information...put what you learned to good use and start coding for gWasm!
```

Author: It was actually copied from the video description. Do you recommend updating it as well?
